transcom/mymove
# B 20599 customer portal button int

## [B-20559](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A981899)

## Summary

This PR just changes the labels for the customer's editing buttons to make their purpose more clear.
This also adds back in the correct version of the mobile home shipment example that was accidentally overwritten here: https://github.com/transcom/mymove/pull/13923/files

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Create any move as a customer
2. Click on "review request"
3. Confirm the ability to edit customer orders
4. Confirm the label change of the uploading doc buttons.

### Frontend


## Screenshots
Button Label changes
<img width="984" alt="Screenshot 2024-10-17 at 2 12 50 PM" src="https://github.com/user-attachments/assets/440c2ccc-f76a-4991-a6c7-cbcec638579e">

Customer Editing orders:
<img width="1063" alt="Screenshot 2024-10-17 at 2 13 43 PM" src="https://github.com/user-attachments/assets/921ed3d3-7537-4c5a-a4fb-fe063b78c874">
